coffee/chosen.jquery.coffee

```diff
@@ -372,7 +372,10 @@ class Chosen extends AbstractChosen
       else
         this.single_set_selected_text(this.choice_label(item))
 
-      unless @is_multiple && (!@hide_results_on_select || (evt.metaKey or evt.ctrlKey))
+      if @is_multiple && (!@hide_results_on_select || (evt.metaKey or evt.ctrlKey))
+        @search_field.val("")
+        @winnow_results()
```

Comment:
Our more-or-less convention prescribes that functions should be accessed through `this`, instead of `@`.

So `this.winnow_results()` in this case.